syft/serde/msgpack/serde.py

```diff
@@ -375,6 +373,14 @@ def deserialize(binary: bin, worker: AbstractWorker = None) -> object:
     return _deserialize_msgpack_simple(simple_objects, worker)
 
 
+def _simplify_field(obj):
```

Comment:
Could you explain what you're doing here?

Reply:
msgpack can't pack python integers larger than 2**64-1 and causes an error like: "Python int too large and can't be converted to C unsigned long long". So we need to convert all values larger than that into string for successful serialisation.